src/components/FallbackNotice.astro

```diff
@@ -1,9 +1,73 @@
 ---
 import Aside from './Aside.astro';
 import UIString from './UIString.astro';
+import { useTranslations } from '../i18n/util';
+const t = useTranslations(Astro);
 ---
 
-<Aside>
-	<UIString key="fallbackContent.notice" />
-	<a href="https://github.com/withastro/docs/blob/main/src/i18n#readme"><UIString key="fallbackContent.linkText" /></a>
-</Aside>
+<div class="fallback-notice">
+	<input type="checkbox" id="fallback-notice-close-checkbox" class="sr-only" aria-hidden="true">
```

Comment:
The close button when focused is being read as "blank". I removed aria-hidden and added the button role to the input and now it is read as "Close, button". I'm not 100% sure if `role="button` is necessary, but I think it's better than "Close, checkbox, not checked"
```suggestion
	<input type="checkbox" id="fallback-notice-close-checkbox" class="sr-only" role="button">
```